docs/frontend/env-setup.mdx

```diff
@@ -4,21 +4,32 @@ sidebar_position: 1
 
 # Environment Setup
 
-Frontend devs should have the following installed and configured as described below:
+Frontend developers should have the following installed and configured as described below:
 
-## NVM (Node version manager)
+## Installing NVM (Node version manager)
 
 :::danger
-Do not install node.js outside of nvm. if you have node.js installed,
-you should uninstall it before proceeding with the nvm install
+Do not install Node.js outside of NVM. if you have Node.js installed,
+you should uninstall it before proceeding with the NVM install
 :::
 
+To install NVM, follow the instructions listed [here](/docs/getting-started/node-nvm)
 
-## Yarn v1
 
-Install yarn from [here](https://classic.yarnpkg.com/en/docs/install)
+## Installing Yarn v1
+:::danger
+Make sure that you have installed Node.js via NPM before proceeding!
```

Comment:
Should be changed to `installed Node.js via nvm`

Reply:
hater